src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Avoid fixing the size of containers if it's not a requirement. If the stars need to be of a specific size, consider making them scalable using relative units like 'em' or 'rem' instead of 'px' to maintain relative sizing and better responsiveness.